ui/js/component/fileCard/view.jsx

```diff
@@ -8,6 +8,23 @@ import UriIndicator from "component/uriIndicator";
 
 class FileCard extends React.PureComponent {
   componentWillMount() {
+    const autoThumbClasses = [
```

Comment:
This behavior should be standardized and used by at least FileTile (and FileShow? not sure about this). Probably involves making `card__media` into a component.